test/assets/isolated-network.xml

```diff
@@ -0,0 +1,9 @@
+<network>
+  <name>${VM_ISOLATED_NETWORK}</name>
```

Comment:
Does the network name need to change?

Reply:
The idea was to have this name defined in `commons.sh` and propagate everywhere. 
It does not need to change at the moment.

Reply:
Makes sense.